### PR TITLE
Upgrade docs

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

**Comment by osisoft-mbishop** [comment]
Hi Dave and Keaton,

On this doc review, I'm looking for two things:
 
1. That the document steps are correct for each OS. If I made any obvious mistakes, please call them out.
2. Any feedback for issues that you discover once the simulator is turned back on.

File: shared-content/installation/upgrade-the-adapter.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Download the latest Windows .msi file from the [OSIsoft Customer Portal](https://customers.osisoft.com/s/products).
+
+    **Note:** Customer login credentials are required to access the portal.
+
+2. Run the .msi file.
+
+3. Follow the setup wizard.
+
+    You can change the installation folder or port number during upgrade. The default port number is `5590`.
```
Comment (osisoft-mbishop) [inline shared-content/installation/upgrade-the-adapter.md:21]: One question about port number--if you change the port number during upgrade, will the user have to manually update their configs?

File: shared-content/installation/upgrade-the-adapter.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   Upon successful upgrade, the JSON response lists the updated application version:
+
+   ```json
+   {
+    "Application Version": "1.1.0.128",                 // upgraded version
+    ".Net Core Version": ".NET Core 3.1.15",
+    "Operating System": "Microsoft Windows 10.0.19041"
+   }
+   ```
+
```
Comment (osisoft-mbishop) [inline shared-content/installation/upgrade-the-adapter.md:38]: One final question: will users lose any data during upgrade?

File: shared-content/installation/upgrade-the-adapter.md (hunk 1 above)
Comment (dhollebeek) [reply shared-content/installation/upgrade-the-adapter.md:21]: Not sure.  Is this allowed on upgrade?  I haven't managed to run through the upgrade process myself yet.

File: shared-content/installation/upgrade-the-adapter.md (hunk 2 above)
Comment (dhollebeek) [reply shared-content/installation/upgrade-the-adapter.md:38]: No.

File: shared-content/installation/upgrade-the-adapter.md (hunk 1 above)
Comment (osisoft-kclause) [reply shared-content/installation/upgrade-the-adapter.md:21]: The upgrade process does not prompt for a port or install directory, but a fresh install does.

Comment (osisoft-mbishop) [reply shared-content/installation/upgrade-the-adapter.md:21]: Thanks for pointing that out @osisoft-kclause. I'll remove it from the upgrade doc.

**Pull request merged**